src/frontend/src/components/alias/test.ts

```diff
@@ -1,10 +1,13 @@
-import { promptDeviceAliasPage } from "./alias";
+import { promptDeviceAliasPage } from "../alias";
 import { I18n } from "../../i18n";
 
 test("can be canceled", async () => {
   const cancel = jest.fn();
-  const ctn = jest.fn((str) => {});
+  const ctn = jest.fn((_str) => {
```

Comment:
the filename changed and eslint is now picking up this file

Reply:
Does this mean we should change the eslint config to be less picky about the filename?